1.9/administration/securing-your-cluster.md

```diff
@@ -35,37 +35,11 @@ By default, Admin Router will permit unencrypted HTTP traffic. This is not
 considered secure, and you must provide a valid TLS certificate and redirect
 all HTTP traffic to HTTPS in order to properly secure access to your cluster.
 
-Once you have a valid TLS certificate, install the certificate on each master.
+After you have a valid TLS certificate, install the certificate on each master.
 Copy the certificate and private key to a well known location, such as under
-`/etc/ssl/certs`. It's suggested that you change the permissions of the
-private key to not be world-readable. Edit the Admin Router `nginx.conf`, by
-opening the file located at
-`/opt/mesosphere/active/adminrouter/nginx/conf`. Modify the nginx
-configuration `server` section, to look like this:
-
-```
-  server {
-    listen 80 default_server;
-    server_name _;
-    return 301 https://$host$request_uri;
-  }
-
-  server {
-      listen 443 ssl spdy default_server;
-      ssl_certificate /etc/ssl/certs/your-cert.crt;
-      ssl_certificate_key /etc/ssl/certs/your-key.key;
-
-      # rest of config intentionally omitted
-```
-
-In the block above, we're adding a new `server` section above the existing one
-which will redirect all HTTP traffic to HTTPS. We've also removed the
-`listen 80 ...` directive from the second server section. After making the
-changes, restart Admin Router:
-
-```console
-# systemctl restart dcos-adminrouter
-```
+`/etc/ssl/certs`. 
+
+It's recommended that you run HAProxy in front of Admin Router. For more information, see the [documentation](/docs/1.9/administration/haproxy-adminrouter/).
```

Comment:
I do not agree with this statement. We do not _generally_ recommend running an HTTP proxy in front of Admin Router.

Also, this changeset suggests an idea (removing instructions for setting up an Admin Router server certificate, and instead adding the recommendation to run HAProxy in front of Admin Router) I do not agree with.

At a high level, that's the view we need to communicate: _if_ one has to run an HTTP proxy (e.g. HAProxy) in front of Admin Router, one _should_ secure the communication between both. For securing Admin Router <-> HAProxy communication, one _has to_ take control of the server certificate presented by Admin Router (Enterprise is doing that while Open is not -- that's why we need corresponding docs for Open DC/OS).